src/adagrams.js

```diff
@@ -1,7 +1,137 @@
+import { worker } from "@jest/reporters/build/coverage_worker";
+
+const alphaLetters = {
+  A: 9,
+  N: 6,
+  B: 2,
+  O: 8,
+  C: 2,
+  P: 2,
+  D: 4,
+  Q: 1,
+  E: 12,
+  R: 6,
+  F: 2,
+  S: 4,
+  G: 3,
+  T: 6,
+  H: 2,
+  U: 4,
+  I: 9,
+  V: 2,
+  J: 1,
+  W: 2,
+  K: 1,
+  X: 1,
+  L: 4,
+  Y: 2,
+  M: 2,
+  Z: 1
+};
+
+const scoreChart = {
+  A: 1,
+  N: 1,
+  B: 3,
+  O: 1,
+  C: 3,
+  P: 3,
+  D: 2,
+  Q: 10,
+  E: 1,
+  R: 1,
+  F: 4,
+  S: 1,
+  G: 2,
+  T: 1,
+  H: 4,
+  U: 1,
+  I: 1,
+  V: 4,
+  J: 8,
+  W: 4,
+  X: 8,
+  K: 5,
+  L: 1,
+  Y: 4,
+  M: 3,
+  Z: 10
+};
+
+const createLetters = function(alphaLetters) {
+  let lettersArray = [];
+  for (let key in alphaLetters) {
+    const result = key.repeat(alphaLetters[key]).split("");
```

Comment:
I really like that you store the letter frequencies and then build the pool on the fly - this is much easier to read than a giant array full of 9 'A's, 2 'B's, etc.